ihatemoney/templates/invitation_mail.fr.j2

```diff
@@ -1,11 +1,11 @@
-Salut,
+Bonjour,
 
-Quelqu'un avec l'adresse "{{ g.project.contact_email }}" vous à invité à partager vos dépenses pour "{{ g.project.name }}".
+Quelqu'un dont l'adresse email est {{ g.project.contact_email }} vous a invité à partager vos dépenses pour « {{ g.project.name }} ».
 
-C'est aussi simple que de dire qui à payé pour quoi, pour qui, et combien celà à coûté, on s’occupe du reste.
+Il suffit de rentrer qui a payé pour quoi, pour qui, et combien ça a coûté, et on s’occupe du reste.
```

Comment:
I think there is one "et" too many : "Il suffit de rentrer qui a payé pour quoi, pour qui, combien ça a coûté, et on s’occupe du reste."

Reply:
indeed :+1:

Reply:
I would prefer a more obvious verb.
```suggestion
Il suffit de renseigner qui a payé pour quoi, pour qui, et combien ça a coûté, et on s’occupe du reste.
```

Reply:
> I would prefer a more obvious verb.

Good idea, done